gamemode/index.js

```diff
@@ -305,13 +297,17 @@ sdk.on("gamemodeLoaded", () => {
     }, 1000);
 });
 
+sdk.on("gamemodeUnloading", () => {
+    console.log("[GAMEMODE] Gamemode unloading!");
+});
+
 sdk.on("playerConnected", (player) => {
     console.log(`[GAMEMODE] Player ${player.getNickname()} connected!`);
     player.sendChatToAll(`[SERVER] ${player.getNickname()} has joined the session!`);
 
-    // player.addWeapon(20, 200); // TODO: Not working yet
-    player.setPosition(sdk.Vector3(-989.397, -289.772, 2.805));
-    player.sendChat(`[SERVER] Welcome ${player.getNickname()}!`)
+    player.addWeapon(2, 200); // doesn't works
+    player.setPosition(sdk.Vector3(-989.397, -289.772, 2.805)); // doesn't works
```

Comment:
Please comment the whole line, and add a TODO comment thus we can grep over all left TODOs

Reply:
I was the one who commented addWeapon earlier but in the end, it's not a problem to keep it uncommented.

However, I'm going to add the TODO!